static/src/stylesheets/module/atoms/_storyquiz.scss

```diff
@@ -0,0 +1,166 @@
+        padding: $gs-baseline / 2 $gs-gutter / 4 $gs-baseline;
+        display: block;
+        font-weight: bold;
+        cursor: pointer;
+        &:hover, &:active {
+            background-color: $neutral-4;
+        }
+    }
+
+    input[type=radio] {
+        margin: 5px;
+    }
+}
+
+.storyquiz__next {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+
+    .button + .button {
+        margin-top: $gs-baseline / 2;
+    }
+}
+
+.storyquiz__card--question:not(.is-answered) .storyquiz__question-result,
+.storyquiz__card--question.is-answered .storyquiz__question,
+.storyquiz__question-comment:not(.is-answer),
+.storyquiz__result:not(.is-result) {
+    display: none;
+}
+
+.explainer-snippet--storyquiz {
+
+    .button--show-more {
+        margin: 24px 0px 12px;
+        font-size: 14px;
+        padding-left: 15px;
+        padding-right: 7px;
+        .inline-icon__svg {
+            height: 35px;
+            width: 30px;
+            position: static;
+        }
+    }
+
+    border-top: 1px solid $news-main-2;
+
+    .explainer-snippet__header {
+        font-family: $f-serif-headline;
+    }
+
+    .explainer-snippet__label {
+        font-weight: 900;
+        font-size: 20px;
+        line-height: 24px;
+
+    }
+
+    .explainer-snippet__body {
+        font-family: $f-serif-text;
+        overflow: visible;
+    }
+
+    .storyquiz__start {
+        font-family: $f-sans-serif-text;
+        font-size: 14px;
+        border-radius: 18px;
+    }
+
+    .explainer-snippet__headline::after {
+        display: inline-block;
+        height: 21px;
+        margin-left: 10px;
+    }
+
+    .is-correct .explainer-snippet__headline::after {
+        content: url('data:image/svg+xml;base64,PHN2ZyBpZD0iTGF5ZXJfMSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2aWV3Qm94PSIwIDAgMjEuOSAyMSI+PHN0eWxlPi5zdDB7ZmlsbDojODdCRTI3O308L3N0eWxlPjxwYXRoIGNsYXNzPSJzdDAiIGQ9Ik0wIDYuM2guMmMwLS42LjYtLjcuOS0xLjEuMi0uMy41LS40LjgtLjIuNi40IDEuMi43IDEuOCAxIC41LjIuNi42LjggMSAuNC43IDEgMS40IDEuMSAyLjIgMCAuMi4xLjUuMS43di4yYy4yLjQuNC45LjYgMS4yLjMuNS45LjQgMS4zIDAgLjYtLjcgMS4zLTEuMyAxLjktMiAuNi0uNiAxLjEtMS40IDEuNi0yIC41LS42IDEuMS0xIDEuNi0xLjYuNS0uNS45LTEuMSAxLjQtMS43LjQtLjUuOC0uOSAxLjMtMS40LjQtLjQuOC0uOSAxLjItMS4zLjMtLjMuNy0uNiAxLS45LjEtLjEuMi0uMS4yLS4xLjQtLjUuOC0uMyAxLjEgMCAuNi41IDEuNC44IDEuOCAxLjYuMy42LjggMS4xIDEgMS43LjMuNC4yIDEuMS0uMSAxLjQtLjcuOS0xLjYgMS43LTIuNCAyLjYtLjUuNS0xIDEuMS0xLjQgMS42LS41LjYtMSAxLjMtMS41IDEuOS0uNi43LTEuMyAxLjMtMiAxLjktLjEuMS0uMi4zLS4zLjQtLjguNy0xLjQgMS42LTIuMiAyLjQtLjUuNi0xLjEgMS4yLTEuNiAxLjgtLjcuOS0xLjQgMS45LTIuMSAyLjgtLjEuMi0uNC4yLS41LjMtLjEuMS0uMy4yLS40LjNoLS4xYy0uNy0uMi0xLjQtLjUtMi4xLS44LS4yLS4xLS40LS40LS42LS42LS4yLS4yLS4zLS40LS40LS42LS4xLS4yLS4yLS40LS4yLS42LS4xLS4yLS4xLS40LS4yLS42LS4yLS4zLS42LS42LS41LTEgMC0uOS0uNC0xLjYtLjctMi40LS4yLS43LS41LTEuNC0uOC0yLjEtLjQtMS4xLS43LTIuMS0xLjEtMy4ydi0uM2MwLS4xLjEtLjMgMC0uNEMuNCA3LjkuMyA3LjMuMiA2LjdjMC0uMS0uMS0uMi0uMi0uM3YtLjF6Ii8+PC9zdmc+');
```

Comment:
Could you please use the inlineSVG helper in the template for this and then only toggle the visibility with CSS?